applications/ray/kuberay-tpu-webhook/Makefile

```diff
@@ -32,12 +32,15 @@ docker-build:
 # Push the docker image  
 docker-push:  
 	docker push ${IMG}  
-  
+
+# Install once per cluster
 install-cert-manager:  
-	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.13.2/cert-manager.yaml
+	helm repo add jetstack https://charts.jetstack.io
```

Comment:
Maybe we should explicitly document this as a prerequisite in the user guide instead of here. Also add explanations for why the user needs to install their own cert manager.

Reply:
I'll add it as a pre-requisite for each cluster and an explanation in the TPU_GUIDE. Should I remove the `make` commands then and just leave the install commands in the guide?